goober.d.ts

```diff
@@ -5,7 +5,7 @@ export = goober;
 export as namespace goober;
 
 declare namespace goober {
-    function styled<T = {}>(tag: string): Tagged<T>;
+    function styled<T = {}>(tag: string | StyledVNode<any>): Tagged<T & { children: any }>;
```

Comment:
From https://github.com/DefinitelyTyped/DefinitelyTyped/pull/32843, they detect if the styled component is a React element. Since this library references React and Preact, should we explicitly be checking if the element is a React element?

Reply:
Thank you! 🙏 And I think it's a good call to not tie `goober` to either of them.